content/docs/guides/crosswalk/kubernetes/apps.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Deploy Kubernetes Apps
+title: Kubernetes App Deployment
```

Comment:
@cnunciato WDYT of this? I'm not sure we should change the title.

Reply:
@zchase Agreed, good catch -- we should not change the title here. The rest are verbs in the nav and it should stay that way IMO.

![image](https://user-images.githubusercontent.com/274700/81985843-acf7de00-95eb-11ea-8851-82988157fb42.png)

Reply:
```suggestion
title: Deploy Kubernetes Apps
```